grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java

```diff
@@ -272,126 +271,27 @@ void sendQueriesToLoader(Collection<Query> queries){
                 .withWaitStrategy(WaitStrategies.fixedWait(1, TimeUnit.SECONDS))
                 .build();
 
-        try {
```

Comment:
I deleted a lot of logic for making sure the number of batches in flight is X. I don't see why we need to check that explicitly. We can just submit a job in a fixed size thread pool that enforces that limit

Reply:
And thank god for that. I was looking at all thise on friday and thinking to myself that it seemed far too complex for something sooooo simple